### PR TITLE
Remove ClientErrors interface from common

### PR DESCRIPTION
**Pull request by DylanTinianov**

### CODE REVIEW

File: common/config/clienterrors.go
Comment (dimriou) [inline common/config/clienterrors.go:1]: I don't think this is the right way to do this. These are evm-specific categories that live inside evm-specific error file.

Comment (DylanTinianov) [reply common/config/clienterrors.go:1]: Good point, removed client errors from common completely.

Comment (dimriou) [inline common/config/clienterrors.go:1]: Is this file needed?

Comment (DylanTinianov) [reply common/config/clienterrors.go:1]: Oops meant to delete it!

**Review by dimriou — APPROVED** [review]

**Review by prashantkumar1982 — APPROVED** [review]
